wrappers/matlab/format.m

```diff
@@ -27,6 +27,7 @@
 		invi     		(24)
 		w10     		(25)
         FG              (26)
-        count           (27)
+        Y411            (27)
```

Comment:
Is this right? In `Format.cs` above is says `30`...

Reply:
done